### PR TITLE
Updating openstack-authorization & response-messaging

### PR DESCRIPTION
**Pull request by usnavi**
Adding response-messaging service configuration
Updating openstack-authorization to support repose 7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -458,6 +459,33 @@ Example hash of inserting headers into responses:
 }
 ```
 
+## response-messaging attributes
+
+* `node[ 'repose' ][ 'response-messaging' ][ 'status_codes' ] - Array of status codes to activate on and generate responses.
```
Comment (filler) [inline README.md]: You might want to clean up any `Space inside square brackets detected.` infractions that rubocop declares.  Even though this is a block in markdown, its consistent elsewhere.

File: templates/default/client-auth-n.cfg.xml.erb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <client-auth xmlns="http://docs.openrepose.org/repose/client-auth/v1.0">
 <% end %>
     <% if @auth_provider == 'RACKSPACE' %>
-<% if @version.split('.')[0].to_i >6 %>
+<% if !@version.nil? && @version.split('.')[0].to_i >6 %>
```
Comment (filler) [inline templates/default/client-auth-n.cfg.xml.erb]: Good catch, sir!  :muscle:

File: recipes/service-response-messaging.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#
+# Cookbook Name:: repose
+# Recipe:: service-response-messaging
+#
+# Copyright (C) 2016 Rackspace Hosting
+#
+
+template "#{node[ 'repose' ][ 'config_directory' ]}/response-messaging.cfg.xml" do
```
Comment (filler) [inline recipes/service-response-messaging.rb]: You might want to clean up any `Space inside square brackets detected.` infractions that rubocop declares.  Consistency ftw.

**Comment by filler** [comment]
Youll probably also want to bump the version here (likely `3.2.0`).

This is really good stuff, @usnavi!  Thanks!

With a clean clone, you can let `rubocop` do most of the fixing via a `rubocop -a`.  Review state via a `git diff`, ensure you work as expected with a `kitchen converge` or a `kitchen test`, and then you can merge and chill.

**Comment by filler** [comment]
```
[cookbook-repose] rubocop -a                                                                                                    15:20:18  ☁  openstack_and_response_messaging ☀

...

55 files inspected, 501 offenses detected, 499 offenses corrected
```

If you find that it does brick your recipes because it was too egregious, you can pick offenses for `rubocop` to pass on which were fatal.

**Comment by usnavi** [comment]
@filler - updated the PR.  Ready for another look.

**Comment by filler** [comment]
@usnavi awesome!

**Pull request merged**